.rubocop.yml

```diff
@@ -1,24 +1,15 @@
-require:
-  - rubocop-performance
-  - rubocop-rails
-
 inherit_from:
   - .rubocop_todo.yml
 
+inherit_gem:
+  theforeman-rubocop:
+    - default.yml
+
 AllCops:
-  TargetRubyVersion: 2.3
-  TargetRailsVersion: 5.1
+  TargetRubyVersion: 2.7
   Exclude:
-    - 'Rakefile'
-    - 'doc/plugins/**'
-    - 'vendor/**/*'
```

Comment:
Similar to others: you need to include `vendor` if you override `Exclude` because it's causing test failures. In this particular plugin I think you can leave it out though.